webpack/components/extensions/HostDetails/Cards/ErrataOverviewCard.scss

```diff
@@ -0,0 +1,3 @@
+.pf-c-empty-state {
+  margin-top: 0;
+}
```

Comment:
Need a newline?

Reply:
Looks like none of the other scss files have it, so I'm happy leaving it :)